e2e/actions/l2_challenger_test.go

```diff
@@ -16,122 +17,149 @@ func TestChallenge(t *testing.T) {
-	rt.setupHonestChallenger()
+	rt.setupHonestChallenger1()
 	rt.setupHonestGuardian()
 
 	// bind contracts
 	rt.bindChallengeContracts()
 
+	// submit outputs
+	rt.setupOutputSubmitted()
+
 	// create challenge
-	rt.setupChallenge()
+	rt.setupChallenge(rt.challenger1)
 
 interaction:
 	for {
-		status, err := rt.colosseumContract.GetStatus(nil, rt.outputIndex)
+		status, err := rt.colosseumContract.GetStatus(nil, rt.outputIndex, rt.challenger1.address)
 		require.NoError(rt.t, err)
 
 		switch status {
 		case chal.StatusChallengerTurn:
 			// call bisect by challenger
-			rt.txHash = rt.challenger.ActBisect(rt.t, rt.outputIndex)
-			rt.miner.includeL1Block(rt.t, rt.challenger.address)
+			rt.txHash = rt.challenger1.ActBisect(rt.t, rt.outputIndex, rt.challenger1.address, false)
+			rt.miner.includeL1Block(rt.t, rt.challenger1.address)
 		case chal.StatusAsserterTurn:
 			// call bisect by validator
-			rt.txHash = rt.validator.ActBisect(rt.t, rt.outputIndex)
+			rt.txHash = rt.validator.ActBisect(rt.t, rt.outputIndex, rt.challenger1.address, true)
 			rt.miner.includeL1Block(rt.t, rt.validator.address)
 		case chal.StatusReadyToProve:
-			rt.txHash = rt.challenger.ActProveFault(rt.t, rt.outputIndex, false)
-			rt.miner.includeL1Block(rt.t, rt.challenger.address)
-		case chal.StatusProven:
-			// validate l2 output submitted by challenger
+			rt.txHash = rt.challenger1.ActProveFault(rt.t, rt.outputIndex, false)
+			rt.miner.includeL1Block(rt.t, rt.challenger1.address)
+		case chal.StatusNone:
+			// guardian validates deleted output by challenger is invalid after challenge is proven
 			outputBlockNum := rt.outputOnL1.L2BlockNumber.Uint64()
-			output := rt.challenger.ActOutputAtBlockSafe(rt.t, outputBlockNum)
-			isValid := rt.guardian.ActValidateL2Output(rt.t, output.OutputRoot, outputBlockNum)
-			require.True(rt.t, isValid)
-			rt.txHash = rt.guardian.ActConfirmTransaction(rt.t, big.NewInt(0))
-			rt.miner.includeL1Block(rt.t, rt.guardian.address)
+			isValid := rt.guardian.ActValidateL2Output(rt.t, rt.outputOnL1.OutputRoot, outputBlockNum)
+			require.False(rt.t, isValid, "deleted output is expected invalid but actually valid")
+			break interaction
 		default:
 			break interaction
 		}
 
-		// Check whether the submission was successful
+		// check whether the submission was successful
 		rt.receipt, err = rt.miner.EthClient().TransactionReceipt(rt.t.Ctx(), rt.txHash)
 		require.NoError(rt.t, err)
 		require.Equal(rt.t, types.ReceiptStatusSuccessful, rt.receipt.Status, "failed to progress interactive fault proof")
 	}
 
-	// Check the status of challenge is StatusApproved(7)
-	status, err := rt.colosseumContract.GetStatus(nil, rt.outputIndex)
+	// check output is deleted
+	remoteOutput, err := rt.outputOracleContract.GetL2Output(nil, rt.outputIndex)
+	require.NoError(rt.t, err, "unable to get l2 output")
+	outputDeleted := val.IsOutputDeleted(remoteOutput.OutputRoot)
+	require.True(rt.t, outputDeleted, "invalid output is not deleted")
```

Comment:
In addition, please check if the submitter of output is changed to challenger correctly.
And also do this for the below tests as well.